### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by giacomo892**
Cleaning up some dead code and white spaces

### CODE REVIEW

**Comment by DusKing1** [comment]
why on experimental?

File: Inc/signal.h [hunk 1]
```diff
@@ -14,7 +14,6 @@ extern int adjusted_input;
 extern int newinput;
 extern char inputSet;
 extern char dshot;
-extern char oneshot125;
```
[the remark this answers is not in the document]
Comment (giacomo892) [reply Inc/signal.h:17]: Aren't all the PR supposed to be against the experimental branch?

[the remark this answers is not in the document]
Comment (cruwaller) [reply Inc/signal.h:17]: It is merged to master already
https://github.com/AlkaMotors/AM32-MultiRotor-ESC-firmware/pull/52

[the remark this answers is not in the document]
Comment (giacomo892) [reply Inc/signal.h:17]: See #56

**Pull request closed without merging**